src/Input.js

```diff
@@ -14,6 +14,10 @@ class Input extends React.Component {
   componentDidMount() {
     if (this.props.type === 'select' && !this.props.browserDefault && typeof $ !== 'undefined') {
       $(this.refs.inputEl).material_select();
+      $(this.refs.inputEl).on('change', this._onChange);
+      if (!!this.props.onChange) {
```

Comment:
You should move this to the `this._onChange` so that the existing function is not override.